lib/github-api.test.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  assertEquals,
+} from "jsr:@std/assert@1";
+import { afterEach, beforeEach, describe, it } from "jsr:@std/testing@1/bdd";
+import { GitHubApiImpl, GitHubCommit, GitHubRelease } from "./github-api.ts";
+
+/**
+ * The tests in this file are meant to be like a playground and not true automated tests. 
+ * These functions are setup so you can quickly call functions against real github repositories. Great during development. 
+ * 
+ * The tests will run as long as the environment variable INPUT_GITHUB_TOKEN is set.
+ */
```

Comment:
As comment says, this file is tests meant for development. They were helpful for debugging.